guide/interactions/replying-to-slash-commands.md

```diff
@@ -0,0 +1,331 @@
+# Replying to slash commands
+
+Discord provides users the option to create client-integrated slash commands. In this section you'll be learning how to respond to these commands using discord.js!
+
+::: tip
+You need to have at least one slash command set-up for your application to follow the instructions on this page. If you haven't done that yet, refer to [the previous page](/interactions/registering-slash-commands/).
+:::
+
+
+## Receiving interactions
+
+Every slash command is an `interaction`, so to respond to a command you need to setup an event listener that will execute code when your application receives an interaction:
+
+```js
+client.on('interaction', interaction => {
+	console.log(interaction);
+});
+```
+::: tip
+You can easily adapt the command handler from earlier sections of the guide to work with interactions and thereby organize your commands properly!
```

Comment:
if I remember correctly the earlier parts of the guide will be refactored to use slash commands?